.gitignore

```diff
@@ -1,6 +1,6 @@
 /.bundle/
 /.yardoc
-/Gemfile.lock
+*emfile.lock
```

Comment:
It generates `./gemfiles/rails_4_1.gemfile.lock`. I don't feel strongly either way. I could change it to be the top-level `./Gemfile.lock` and `./gemfiles/*gemfile.lock`